src/effects/VST3/VST3Effect.cpp

```diff
@@ -1235,24 +1011,11 @@ void VST3Effect::ReloadUserOptions()
 
 EffectSettings VST3Effect::MakeSettings() const
 {
-   auto result = StatefulPerTrackEffect::MakeSettings();
-   // Cause initial population of the map stored in the stateful effect
-   if (!mInitialFetchDone) {
-      FetchSettings(GetSettings(result));
-      mInitialFetchDone = true;
-   }
-   return result;
```

Comment:
Last things to say about the fifth commit.
When I transformed other effect families, the move to actual statelessness came late.  Until a late commit, `GetSettings` was a non-static member function, ignoring its argument, and really using information in the stateful Effect object.

All the places where `GetSettings` had to be called were figured out first; and only then did I make a change so that the `EffectSettings` objects really carried independent information around and were not dummies.

You make this change earlier, so `EffectSettings` objects really carry data, making it possible that buggy behavior exists in the intermedate steps of this transformation because of inconsistencies about where the correct data really are.  I already mentioned the possible wrong behavior of exporting.

It's not how I would write a very careful transoformation.  In case bugs still exist after everything and I want to bisect to understand where the bug was introduced, the other caution I took with intermediate steps might help me find out where the mistake was really introduced.

I'm not insisting now on rewriting this pull request that way.